### ISSUE TITLE
 fix `publish-ghcr.yml` workflow version number

### ISSUE THREAD

**Issue by Pradumnasaraf** [Pradumnasaraf/Botchain#20]
### Description

Update the 3 actions to the latest version

```diff
- uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+ docker/login-action@v2
```

```diff
- uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+ docker/metadata-action@v4
```

```diff
- uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
+ docker/build-push-action@v4
```

File path -https://github.com/Pradumnasaraf/Botchain/blob/main/.github/workflows/publish-ghcr.yml

**Comment by tarunsamanta2k20** [comment]
I would like to work on this issue

**Comment by Pradumnasaraf** [comment]
> I want to work on this

Yes go ahead

**Comment by Pradumnasaraf** [comment]
fix in #24
